sql/views.sql

```diff
@@ -25,6 +25,23 @@ CREATE OR REPLACE VIEW timescaledb_information.hypertable_size_info AS
       CASE WHEN has_schema_privilege(ht.schema_name,'USAGE') THEN format('%I.%I',ht.schema_name,ht.table_name) ELSE NULL END
     ) bsize ON true;
 
+-- View to provide some information about chunks
+-- chunk_id     Chunk ID
+-- lower_bound  Lower time constraint bound for chunk
+-- upper_bound  Upper time constraint bound for chunk
+-- hypertable   Hypertable that the chunk belongs to
+CREATE OR REPLACE VIEW timescaledb_information.chunks AS
+SELECT format('%1$I.%2$I', ch.schema_name, ch.table_name)::regclass AS chunk_id
```

Comment:
These two IDs have confusing names given that we have internal IDs too. Should probably be something like chunk_oid or cunk_relid.

Reply:
I actually think it the name is fine (there is no risk for confusion and it happen to be a reference to a chunk), but it could be something like `chunk_id` since it is just an identifier and any connection to other tables is a coincidence. If we use `regclass` it is an OID, but if we decide for some other representation, we will have to change the name again. Not sure if we have any agreement on this.

Called it `_oid` in the meantime since that seem to be close to what we discussed last. We can change later if we want.